foreman/foreman.spec

```diff
@@ -424,6 +436,8 @@ cp config/settings.yaml.example config/settings.yaml
 export BUNDLER_EXT_NOSTRICT=1
 export BUNDLER_EXT_GROUPS="default assets"
 %{scl_rake} assets:precompile:all RAILS_ENV=production --trace
+%{scl_rake} db:migrate RAILS_ENV=production --trace
```

Comment:
I think this also needs to be in the RPM macros so plugins also migrate during build.  The foreman_discovery build fails right now as it can't open the DB while running plugin:apipie:cache.